pkg/quarterdeck/db/models/users_test.go

```diff
@@ -524,3 +525,69 @@ func (m *modelTestSuite) TestUserPermissions() {
 	require.NoError(err, "could not fetch permissions for user")
 	require.Len(permissions, 18, "wrong number of permissions, have the owner role permissions changed?")
 }
+
+func (m *modelTestSuite) TestListUsers() {
+	require := m.Require()
+
+	ctx := context.Background()
+	orgID := ulid.MustParse("01GQFQ14HXF2VC7C1HJECS60XX")
+
+	//test passing null orgID results in error
+	users, cursor, err := models.ListUsers(ctx, ulids.Null, nil)
+	require.ErrorIs(err, models.ErrMissingOrgID, "orgID is required for list queries")
+	require.NotNil(err)
+	require.Nil(cursor)
+	require.Nil(users)
+
+	_, _, err = models.ListUsers(ctx, orgID, &pagination.Cursor{})
+	require.ErrorIs(err, models.ErrMissingPageSize, "pagination is required for list queries")
+
+	// Should return all checkers users (page cursor not required)
+	users, cursor, err = models.ListUsers(ctx, orgID, nil)
+	require.NoError(err, "could not fetch all users for checkers org")
```

Comment:
Guessing that the orgID `01GQFQ14HXF2VC7C1HJECS60XX` corresponds to a test fixture for an org called Checkers?

Reply:
yes - I modified the comments to make this more clear